daemon/daemon_test.go

```diff
@@ -218,68 +218,15 @@ func (s *daemonSuite) TestMaintenanceJsonDeletedOnStart(c *check.C) {
 
 	b, err := json.Marshal(maintErr)
 	c.Assert(err, check.IsNil)
-
 	c.Assert(os.MkdirAll(filepath.Dir(dirs.SnapdMaintenanceFile), 0755), check.IsNil)
-
 	c.Assert(ioutil.WriteFile(dirs.SnapdMaintenanceFile, b, 0644), check.IsNil)
 
 	d := newTestDaemon(c)
+	makeDaemonListeners(c, d)
 
-	// mark as already seeded
```

Comment:
This removes a bit more than makeDaemonListeners implements (no "core" snap mocking in the state), I assume this was irrelevant for the test?

Reply:
Yes this test is really just about deleting the maintenance.json file